pom.xml

```diff
@@ -690,6 +690,16 @@
                 <artifactId>javax.el-api</artifactId>
                 <version>3.0.0</version>
             </dependency>
+            <dependency>
+                <groupId>com.squareup.retrofit</groupId>
+                <artifactId>retrofit</artifactId>
+                <version>2.0.0-beta2</version>
```

Comment:
Do we need to use a beta version of retrofit? Probably there is a good reason for it, but I just want to check :smile: